testing/testing.go

```diff
@@ -129,20 +128,19 @@ func cd(t *testing.T, dir string) {
 	}
 }
 
-// TestRepoURI starts serving HTTP git server with GIT_PROJECT_ROOT=$(pwd)/testdata
-// and returns URL for project named `name` under the git root.
-//
-// For example TestRepoURI("my-repo", t) returns string that could look like:
-// http://localhost:4242/my-repo.git
-func TestRepoURI(name string, t *testing.T) string {
+// ServeRepositry [name] from ./testdata/[name] returning URL at which
+// the named repository is available.
+// Must be called before any helpers which change test working directory
+// such as fromTempDirectory(t)
+func ServeRepo(name string, t *testing.T) string {
```

Comment:
This seemed like a _slight_ naming improvement since it underscores that the call actually starts the server as well as returns the address.